### ISSUE TITLE
Verify evals on Papers with Code

### ISSUE THREAD

**Issue by NielsRogge** [yermandy/deepfake-detection#16]
Hi,

Niels here from the open-source team at Hugging Face. Congratulations on your work!

I've made the [paper](https://paperswithcode.co/paper/2503.19683) and [1 paper-native evaluation](https://paperswithcode.co/paper/2503.19683#results) available on Papers with Code.

The paper is part of the [Deepfake Detection](https://paperswithcode.co/tasks/deepfake-and-forensics) task page.

The CLIP ViT-L/14 (LN tuning) result currently ranks second on [FFIW-10K (FF++→FFIW Cross-Dataset Video AUROC)](https://paperswithcode.co/benchmark/ffiw-10k-ff-ffiw-cross-dataset-video-auroc?task=deepfake-and-forensics&eval=20249).

Would it be possible to verify these results and let me know if any score, model name, benchmark protocol, or openness metadata should be corrected? The imported rows are tied to the paper or its official release artifacts; comparison-table baselines were not added.

You can also edit the task, methods, project page, and GitHub URL directly from the paper page using your Hugging Face account.

If you'd like to showcase the results in your repository README, you can copy these live leaderboard badges (or use the “Copy PwC badge” button in the Results section):

[![Papers with Code: #2 on FFIW-10K (FF++→FFIW Cross-Dataset Video AUROC)](https://paperswithcode.co/api/v1/papers/2503.19683/leaderboard-badge.svg?eval=20249&live=1)](https://paperswithcode.co/api/v1/papers/2503.19683/leaderboard-badge-link?eval=20249)

Kind regards,

Niels

**Comment by yermandy** [comment]
Thanks for your work, @NielsRogge; this looks really good

**Comment by NielsRogge** [comment]
Thanks for checking!

Let me know how I can further improve https://paperswithcode.co/tasks/deepfake-and-forensics.

Also, feel free to copy the badge in your README ;)

**Comment by yermandy** [comment]
@NielsRogge, to be honest, deepfake detection is not as standardized as lots of known ImageNet-like / COCO-like, etc. benchmarks. There are a lot of datasets that I would like to see in https://paperswithcode.co/tasks/deepfake-and-forensics but the community converged on a standard cross-dataset protocol: training on the FF++ and evaluating on the rest. I do not think that in 2026 this protocol is still good enough, as we are entering the age of entire face synthesis datasets.  

What I want to say is the following: reported scores should be reported with training data; in-distribution performance is too easy in this task; cross-dataset performance is hard. Yet, FF++ cross-dataset protocol seems to be quite obsolete to me. 

Besides all this, there are problems with reproducibility. Reported scores sometimes cannot be reproduced on the same datasets due to a few reasons: frame-based models usually process frames independently, and the scores are then averaged. The common setting is sampling 32 frames uniformly, but then there is a variation in which frames were sampled, causing a variation in reported-reproduced scores. Sometimes, people also report frame-level AUROC, which is even worse on my mind, causing even bigger variance as some methods sample 8/16/32 frames.

**Comment by yermandy** [comment]
At the very least, I would include DeepSpeak dataset (CVPR 2026), comes in two versions:
1.1: https://huggingface.co/datasets/faridlab/deepspeak_v1_1
2.0: https://huggingface.co/datasets/faridlab/deepspeak_v2

paper: https://arxiv.org/abs/2408.05366

**Comment by NielsRogge** [comment]
Hi @yermandy,

Thanks, this is exactly the kind of protocol distinction the catalog should preserve.

I've updated the [Deepfake Detection task](https://paperswithcode.co/tasks/deepfake-and-forensics) description to make the training data and regime, dataset version, modality, frame- versus video-level granularity, frame-sampling count, score aggregation, and metric explicit. It now also states that frame-level and video-level results are not directly comparable.

I've also added the two versions separately:

- [DeepSpeak v1.1](https://paperswithcode.co/benchmark/deepspeak-v1-1)
- [DeepSpeak v2.0](https://paperswithcode.co/benchmark/deepspeak-v2-0)

Both are tied to [The DeepSpeak Dataset](https://paperswithcode.co/paper/2408.05366) and their official Hugging Face releases. I did not pool the paper's comparison-table baselines into a leaderboard because the current table does not unambiguously assign every result to one dataset version. Future results should keep external-pretrained or cross-dataset, DeepSpeak-trained, and fine-tuned regimes distinct, together with their sampling and aggregation details.

Thanks again for the thoughtful guidance!

**Comment by yermandy** [comment]
I want to add results. I need to define a metric first. What are Task/Benchmark ID?

<img width="441" height="481" alt="Image" src="https://github.com/user-attachments/assets/4bab53d2-999d-4883-a0b5-423d587e2bea" />

**Comment by NielsRogge** [comment]
Hi @yermandy thanks for reporting that, I'll get it fixed!

Normally you should link a task and benchmark. I'll fix it
